src/components/Checkbox/Checkbox.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styles from "./Checkbox.module.scss";
+import { color } from "@storybook/addon-knobs";
```

Comment:
seems not used in the file - remove it please

Reply:
The styles will be now used, but I'll remove rest.